### PR TITLE
Add facebook sing in in the login and signup screens. Unit tests add

### PR DESCRIPTION
**Pull request by jimibilibob**

### CODE REVIEW

File: RoutesApp_ios/Info.plist [hunk 1]
```diff
@@ -32,5 +21,23 @@
...
+            <string>Editor</string>
+            <key>CFBundleURLSchemes</key>
+            <array>
+                <string>com.googleusercontent.apps.11042396908-eocvrc8aj9qnk099m0kbjqdoe3a6d2hc</string>
+                <string>fb3146525082229053</string>
+            </array>
+        </dict>
+    </array>
+    <key>FacebookAppID</key>
+    <string>3146525082229053</string>
```
Comment (jmacboy) [inline RoutesApp_ios/Info.plist]: this should not be added to the main repository

File: RoutesApp_ios/Info.plist [hunk 2]
```diff
@@ -32,5 +21,23 @@
...
+    <key>FacebookClientToken</key>
+    <string>131412692dc6c5687212fa62aa82474a</string>
```
Comment (jmacboy) [inline RoutesApp_ios/Info.plist]: this is secret info

**Review by jmacboy — DISMISSED** [review]

File: RoutesApp_ios/Info.plist (hunk 1 above)
Comment (jimibilibob) [reply RoutesApp_ios/Info.plist]: How could I do it? should I replace the keys???

Comment (jmacboy) [reply RoutesApp_ios/Info.plist]: those keys should be manually setup but they have no need to be on the repository.
Add a placeholder field, like a random number or a string that says YOUR_APP_ID_HERE

File: RoutesApp_ios/Info.plist (hunk 2 above)
Comment (jmacboy) [reply RoutesApp_ios/Info.plist]: also here

**Review by jmacboy — APPROVED** [review]
it says fbFB, but its approved anyway

**Pull request merged**